georocket-server-api/src/main/java/io/georocket/storage/ChunkMeta.java

```diff
@@ -33,6 +34,55 @@ public ChunkMeta(List<XMLStartElement> parents, int start, int end) {
   }
   
   /**
+   * Chunk Meta from JsonObject
+   * @param object The JsonObject containing the values for the ChunkMeta Object.
+   */
+  public ChunkMeta(JsonObject object){
+    JsonArray parentsArr = object.getJsonArray("parents");
+    parents = parentsArr.stream().map(e ->
+        XMLStartElement.fromJsonObject((JsonObject)e)).collect(Collectors.toList());
+    start = object.getInteger("start");
+    end = object.getInteger("end");
+  }
+  
+  /**
+   * Chunk Meta from Map.
+   * Used to convert Elasticsearch hit to ChunkMeta
+   * @param source The Elasticsearch document
+   */
+  @SuppressWarnings("unchecked")
+  public ChunkMeta(Map<String, Object> source){
```

Comment:
The attributes you use here are only known to the concrete IndexerVerticle implementation. This code does not belong here.